.github/workflows/migrations-metadata-deployment.yml

```diff
@@ -0,0 +1,80 @@
+name: "Applies the migrations to the database"
```

Comment:
This needs to be updated, its the same name as https://github.com/cityofaustin/atd-moped/blob/main/.github/workflows/atd_moped_database.yml#L4